blog/custom-cts.md

```diff
@@ -0,0 +1,128 @@
+---
+hide_table_of_contents: true
+title: "Exploring the potential of custom content types"
+date: 2023-12-22
+image: "./media/snow-cover.png"
+authors: eng
+description: "Demonstrating the versatility of XMTP custom content types with a festive example"
+tags:
+  - Content Types
+  - Developers
+  - Messaging
+---
+
+import FeedbackWidget from '/src/components/FeedbackWidget'
+
+### Demonstrating the versatility of XMTP custom content types with a festive example
+
+![let it snow screen effect in xmtp.chat](./media/snow-cover.png)
+
+As we celebrate the holiday season, we’re excited to showcase the flexibility of XMTP with a whimsical winter-themed ScreenEffect in the [xmtp.chat](https://xmtp.chat/) example app. 
+
+<!--truncate-->
+
+This ScreenEffect may be just for fun, but it's a glimpse into the creative possibilities of custom content types more generally.
+
+Interested in how it works? Simply long-press the send button in the app and choose the effect you’d like to run your message with.
+
+![xmtp.chat UI displaying a menu enabling a user to choose to send a message with a "Let it Snow" or "Make it Rain XMTP!" screen effect](media/let-it-snow.png)
+
+Custom content types in XMTP empower developers to enrich their app's capabilities and can ultimately **elevate the entire XMTP ecosystem**.
+
+Differing from standard content types, custom content types like this one don't require formal adoption through the XIP process because they’re still considered “experimental.” 
+
+This opens the door for immediate experimentation and use in your own projects. Creating a custom content type can be a helpful first step toward getting a new content type adopted as an XMTP standard through the [XIP process](https://github.com/xmtp/XIPs).
+
+However, it’s important to remember that custom content types aren't automatically supported across apps. To use someone else's custom content type, you'll need to implement its code in your app. That's why sharing the content type, as we did with this ScreenEffect content type in the [`experimental`](https://github.com/xmtp/xmtp-js-content-types/tree/main/experimental/content-type-screen-effect) directory, is crucial for community collaboration.
+
+## 🔍 Delve into the code
+
+Curious about crafting your own custom content type to share with other XMTP apps? Let's dive into the codec powering [xmtp.chat](https://xmtp.chat/).
+
+Here's a quick glossary of fields that are required for all content types:
+
+- `authorityId`: Identifies the entity that governs the content type, its definition, and implementation. Authority ID should be unique and widely recognized as belonging to the entity. DNS domains or ENS names can serve this purpose (e.g. `alix.eth`). The `xmtp.org` authority ID is reserved for standard content types, as well as this custom content type used for demonstration purposes.
```

Comment:
Ah, I think `xmtp.org` is truly reserved for standard content types only - so only content types that have been through the XIP process, have an XRC in final status, and are available in the SDKs. 

Though I know that we do have some content types that are in the SDKs that don't have XRCs in final status. But this one in particular in experimental status shouldn't use xmtp.org, I think.

Though I say this only because this requirement is stated here in XIP-5: https://github.com/xmtp/XIPs/blob/main/XIPs/xip-5-message-content-types.md#content-type-identifier-and-parameters. 

Any thoughts?

Reply:
Really good catch that it was specifically stated there. i originally had `xmtp.chat` as the authority id but incorporated some PR feedback where i updated it to .org. Thinking of now reverting back to xmtp.chat but would love to get @rygine to weigh in here as well!

Reply:
what about `experimental.xmtp.org` or `xmtp.org/experimental` as the authority id?

Reply:
Thank you, @daria-github and @rygine - I think `experimental.xmtp.org` works. Feels like an accurate representation of the entity that governs it. =)

Reply:
sounds good, just made the update!